Misc/NEWS

```diff
@@ -371,6 +371,8 @@ Library
 - bpo-30068: _io._IOBase.readlines will check if it's closed first when
   hint is present.
 
+- bpo-29447: tempfile's dir argument is now accept path-like objects.
```

Comment:
Usually we don't add entries about changes to tests unless this is very large change or a change that affects the testing infrastructure (test.support, test.libregrtest).